src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -309,6 +311,56 @@ object ProtocPlugin extends AutoPlugin with Compat {
       }
     }
 
+  // Forward lines from an OutputStream to a given logger at a specified level
+  private class LoggingOutputStream(logger: Logger, level: Level.Value) extends OutputStream {
+    private val baos = new ByteArrayOutputStream
+    override def write(b: Int): Unit = {
+      baos.write(b)
+      val lineSoFar = baos.toString
```

Comment:
It looks like we add one byte at a time, but may convert to UTF-8 (or whatever the JVM locale is) without having a complete multibyte character at the end of the byte array. It also seems that this is quadratic comlexity in the line length (we build a new string on every byte).

Reply:
* if the stream is in the middle of a multibyte character, wouldn't the suffix not match anyway? or is your concern that `toString` would fail with an unchecked exception in that case?
* regarding the complexity - agreed, I just went for a naive implementation first, but I can iterate on that

I'll extract this helper class to a separate file and unit-test it.

Reply:
Done in d3a61f1d81cb2e42df44b0e866e38b36d7ebcc99.